### ISSUE TITLE
Подключение описания метаданных для автодополнения

### ISSUE THREAD

**Issue by malikov-pro** [infostart-hub/snegopat#98]
**Существует проблема**
Нет автодополнения при использовании объектов из соседних расширений

**Хотелось бы иметь следующую функциональность**
Подключить файл со списком объектов и их методов для использования для использования в автодополнении.
Файл формировать по папке с исходниками расширений с уточненением типов и струтуры вложенных реквизитов и функций.

**Дополнительный контекст**
С похожим работал в https://plugins.jetbrains.com/plugin/7616-bitrix-framework-support, возможно не лучший пример.

**Comment by artbear** [comment]
Да, давно напрашивается механизм расширения контекстной подсказки.

Хотя бы из файла.

@orefkov @kuntashov @7OH что скажете?

@orefkov возможно, это не так сложно сделать в корневых скриптах на АнгелСкрипт.
подскажешь как?

**Comment by kuntashov** [comment]
@artbear  портировать функционал Intelllisense.vbs? )

Нужно событие отлова всплывающей подсказки и апи для изменения списка подсказки.

Или свой список показывать по отдельному хоткею.
